### PR TITLE
Breadcrumbs with select

### PR DESCRIPTION
**Pull request by pgiraud**
This MR readds the selection menus in the breadcrumb.

### CODE REVIEW

**Comment by pgiraud** [comment]
@rjuju please review.

File: powa/overview.py [hunk 1]
```diff
@@ -80,11 +78,12 @@ def dashboard(self):
...
     @classmethod
     def get_childmenu(cls, handler, params):
+        from powa.server import ServerOverview
         children = []
         for s in list(handler.servers):
             new_params = params.copy()
             new_params["server"] = s[0]
             entry = ServerOverview.get_selfmenu(handler, new_params)
-            entry.title = s[1]
+            entry.title = handler.deparse_srvid(s[0])
```
Comment (rjuju) [inline powa/overview.py]: this adds an O(n) complexity for the breadcrumb.  You could use s[2] and add the necessary information in servers()@framework.py to have the information basically for free, ie:

```diff
diff --git a/powa/framework.py b/powa/framework.py
index 2e46bbc..28dcdfb 100644
--- a/powa/framework.py
+++ b/powa/framework.py
@@ -278,13 +278,14 @@ class BaseHandler(RequestHandler):
         """
         if self.current_user:
             if self._servers is None:
-                self._servers = [[s['id'], s['val']] for s in self.execute(
+                self._servers = [[s['id'], s['val'],s ['alias']] for s in self.execute(
                     """
                     SELECT s.id, CASE WHEN s.id = 0 THEN
                         %(default)s
                     ELSE
                         s.hostname || ':' || s.port
-                    END AS val
+                    END AS val,
+                    s.alias
                     FROM {powa}.powa_servers s
                     ORDER BY hostname
                     """, params={'default': self.current_connection})]
```

File: powa/static/js/components/BreadCrumbs.vue [hunk 2]
```diff
@@ -44,4 +51,8 @@ watch(
     });
   }
 );
+
+function onSelect(url) {
```
Comment (rjuju) [inline powa/static/js/components/BreadCrumbs.vue:55]: I'm a bit afraid it's quite a generic name.  Maybe onBreadcrumbsSelect() or something like that?

**Review by rjuju — CHANGES_REQUESTED** [review]
Looks quite nice thanks!  Just a few minor complaints but it overall looks good.

File: powa/static/js/components/BreadCrumbs.vue (hunk 2 above)
Comment (pgiraud) [reply powa/static/js/components/BreadCrumbs.vue:55]: This function is **only** available in the component context.

Comment (rjuju) [reply powa/static/js/components/BreadCrumbs.vue:55]: Oh then no objection

File: powa/overview.py (hunk 1 above)
Comment (pgiraud) [reply powa/overview.py]: Implemented via https://github.com/powa-team/powa-web/pull/176 instead.

Comment (rjuju) [reply powa/overview.py]: you should have told me you wanted to split that, I could have taken care of it.  Anyway I will merge it.

**Comment by pgiraud** [comment]
I merged this MR, even though there is a bug. The following error when accessing "/config/1".

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/dist-packages/tornado/web.py", line 1711, in _execute
    result = method(*self.path_args, **self.path_kwargs)
  File "/usr/local/src/powa-web-dev/powa/dashboards.py", line 58, in get
    return self.render(self.template,
  File "/usr/local/lib/python3.9/dist-packages/tornado/web.py", line 865, in render
    html = self.render_string(template_name, **kwargs)
  File "/usr/local/lib/python3.9/dist-packages/tornado/web.py", line 1014, in render_string
    return t.generate(**namespace)
  File "/usr/local/lib/python3.9/dist-packages/tornado/template.py", line 362, in generate
    return execute()
  File "fullpage_dashboard_html.generated.py", line 181, in _tt_execute
    last = len(handler.breadcrumb) - 1  # layout.html:189
  File "/usr/local/src/powa-web-dev/powa/dashboards.py", line 116, in breadcrumb
    breadcrumb = self.get_breadcrumb(self, params)
  File "/usr/local/src/powa-web-dev/powa/dashboards.py", line 669, in get_breadcrumb
    entry.children_title = cls.title % params
AttributeError: type object 'RemoteConfigOverview' has no attribute 'title'
```

**Comment by pgiraud** [comment]
Fixed.

**Pull request merged**